frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/EditStepBlock.tsx

```diff
@@ -41,6 +43,8 @@ export const EditStepBlock = ({
   const formMethods = useForm<EditStepInputs>({
     defaultValues,
   })
+  const { user, isLoading: isUserLoading } = useUser()
+  isLoading = isLoading || isUserLoading
```

Comment:
/nit would prefer renaming the props `isLoading: _isLoading` so that we don't lose the passed in props.